handlers/mparticle-api/src/routers/baton.ts

```diff
@@ -0,0 +1,134 @@
+import { v4 as uuidv4 } from 'uuid';
+import type { DataSubjectRequestState } from '../../interfaces/data-subject-request-state';
+import { DataSubjectRequestStatus } from '../../interfaces/data-subject-request-state';
+import type { DataSubjectRequestSubmission } from '../../interfaces/data-subject-request-submission';
+import {
+	getStatusOfDataSubjectRequest,
+	submitDataSubjectRequest,
+} from '../apis/data-subject-requests';
+import { setUserAttributesForRightToErasureRequest } from '../apis/events';
+import { getEnv } from '../utils/config';
+
+// https://github.com/guardian/baton/blob/1037c63c9bd782aed514bf6aaa38a54dabf699eb/README.md
```

Comment:
very handy comment.  Could be useful to reference the correct section.  And mention what they will see . as follows
```suggestion
// baton data structures are described here
// https://github.com/guardian/baton/blob/1037c63c9bd782aed514bf6aaa38a54dabf699eb/README.md#general-request--response-fields
```

Reply:
Improved @johnduffell , thanks 💪